atd-vze/src/styles/colors.js

```diff
@@ -0,0 +1,5 @@
+// Hex values of colors from Bootstrap
+export const colors = {
+  // Bootstrap color: hex value
+  light: "#c8ced3",
```

Comment:
thanks for doing this, starts small but sets a pattern that will pay dividends for our productivity and consistency.

Reply:
👍